infrastructure/scripts/aws/README.md

```diff
@@ -174,4 +176,21 @@ monitor the test.
 ./copy_to_cluster.sh --tag profiling -- /Applications/YourKit-Java-Profiler-2019.1.app/Contents/Resources/bin/linux-x86-64/libyjpagent.so .
 ./run_tests.sh --tag profiling --geode-branch develop -- -i -Pbenchmark.profiler.argument=-agentpath:/home/geode/libyjpagent.so=disablestacktelemetry,exceptions=disable,delay=60000,sessionname=JVM_ROLE-JVM_ID
 ./destroy_cluster.sh --tag profiling
+```
+
+# Running with SSL enabled
+
+## Prerequisites
+* You must have fulfilled the prerequisites at the beginning of this doc
+* Generate a self-signed SSL certificate using the keytool command:
+  * `keytool -genkey -keyalg RSA -alias tomcat -keystore selfsigned.jks -validity <days> -keysize 2048`
```

Comment:
Let's use concrete values for the example. 
`keytool -genkey -keyalg RSA -alias geode -keystore selfsigned.jks -validity 365 -keysize 2048`

Reply:
@pivotal-jbarrett done